### PR TITLE
Add redirects.

### PR DESCRIPTION
**Pull request by Sfshaza**
Fixes https://github.com/dart-lang/site-www/issues/249

Please review, @kwalrath

### CODE REVIEW

File: firebase.json [hunk 1]
```diff
@@ -446,6 +451,11 @@
         "type": 301
       },
       {
+        "source": "/docs/dart-up-and-running/contents/ch02",
+        "destination": "/resources/books",
```
Comment (kwalrath) [inline firebase.json]: Shouldn't the destination be /guides/language/language-tour, just like the next redirect?

**Comment by kwalrath** [comment]
Thanks for doing this! Have you staged this?

File: firebase.json (hunk 1 above)
Comment (filiph) [inline firebase.json]: pinging @Sfshaza

**Comment by Sfshaza** [comment]
Thanks for reminding me about this, @filiph!

**Pull request merged**